schutzbot/include_image_info.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -euxo pipefail
+sudo dnf install gh -y
+sudo pip install python-gitlab
```

Comment:
Have you tried `dnf install python3-gitlab`? Might be better to stick to OS dependencies for the sake of consistency.